src/lib/Behat/PageObject/Page.php

```diff
@@ -62,4 +62,14 @@ public function verifyRoute(): void
     public function verifyElements(): void
     {
     }
+
+    /**
+     * Gets the header text displayed in AdminUI.
+     *
+     * @return string
+     */
+    public function getPageHeaderTitle(): string
```

Comment:
This is up to discussion - I know that it does not belong to all Pages (LoginPage, Studio), but don't want to create a new abstraction layer (AuthenticatedPage or something like this).

Reply:
I think it's good enough. Alternatively we can place it in UpperMenu, but it is equally acceptably-inaccurate.

Reply:
<img width="587" alt="screen shot 2018-01-24 at 13 25 18" src="https://user-images.githubusercontent.com/10993858/35332058-fa8cea08-0109-11e8-99db-454aa45c2d4d.png">
(screenshot for clarity) - it's this title, it returns "Home Landing Page" for this example